packages/connect-thegraph/src/types.ts

```diff
@@ -1,6 +1,14 @@
-import { OperationResult } from "@urql/core"
+import { OperationContext, OperationResult } from '@urql/core'
 
 export type QueryResult = OperationResult<any>
 export type DataGql = any
 
 export type ParseFunction = (data: DataGql, connector: any) => {}
+
+// From https://github.com/FormidableLabs/urql/blob/ca68584a578b9f85d0b1448fc7a5fc9587f968de/packages/core/src/exchanges/subscription.ts#L39-L44
+export interface SubscriptionOperation {
+  query: string
+  variables?: object
+  key: string
+  context: OperationContext
+}
```

Comment:
This is to avoid importing a file using an internal path (`@urql/core/dist/types/exchanges/subscription`), and prevent it to break while also making it possible to follow different import paths based CJS / ESM support.

Reply:
Good choice.